### PR TITLE
Fixing variable dependency

### PR DESCRIPTION
**Pull request by Julian1998**
oc_current_user doesn't exist anymore and has to be replaced by OC.getCurrentUser() for supporting all necessary versions.

### CODE REVIEW

**Review by DeepDiver1975 — APPROVED** [review]

File: apps/files/js/files.js [hunk 1]
```diff
@@ -114,11 +114,12 @@
 				return;
 			}
 
+			var currentUser = OC.getCurrentUser();
```
Comment (DeepDiver1975) [inline apps/files/js/files.js]: wait ..... getCurrentUser returns an array

bildschirmfoto von 2018-02-19 08-51-39

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

**Comment by PVince81** [comment]
Nice catch! Please see the above.

File: apps/files/js/files.js (hunk 1 above)
Comment (Julian1998) [reply apps/files/js/files.js]: Sure! I knew that - But calendar and co. need an object

Comment (Julian1998) [reply apps/files/js/files.js]: Sure! I knew that - But calendar needs this object. oc_current_user and getCurrentUser() are equal!

**Comment by Julian1998** [comment]
Sorry...my fault - Fixed

**Comment by PVince81** [comment]
squashed and rebased

**Pull request merged**